server/src/internalClusterTest/java/org/opensearch/search/simple/SimpleSearchIT.java

```diff
@@ -6,258 +6,24 @@
-        assertHitCount(search, 5L);
-
-        search = client().prepareSearch().setQuery(boolQuery().must(QueryBuilders.termQuery("ip", "192.168.1.5/32"))).get();
-        assertHitCount(search, 0L);
-
-        assertFailures(
-            client().prepareSearch().setQuery(boolQuery().must(QueryBuilders.termQuery("ip", "0/0/0/0/0"))),
-            RestStatus.BAD_REQUEST,
-            containsString("Expected [ip/prefix] but was [0/0/0/0/0]")
-        );
-    }
-
-    public void testSimpleId() {
-        createIndex("test");
-
-        client().prepareIndex("test").setId("XXX1").setSource("field", "value").setRefreshPolicy(IMMEDIATE).get();
-        // id is not indexed, but lets see that we automatically convert to
-        SearchResponse searchResponse = client().prepareSearch().setQuery(QueryBuilders.termQuery("_id", "XXX1")).get();
-        assertHitCount(searchResponse, 1L);
-
-        searchResponse = client().prepareSearch().setQuery(QueryBuilders.queryStringQuery("_id:XXX1")).get();
-        assertHitCount(searchResponse, 1L);
-    }
-
-    public void testSimpleDateRange() throws Exception {
-        createIndex("test");
-        client().prepareIndex("test").setId("1").setSource("field", "2010-01-05T02:00").get();
-        client().prepareIndex("test").setId("2").setSource("field", "2010-01-06T02:00").get();
-        ensureGreen();
-        refresh();
-        SearchResponse searchResponse = client().prepareSearch("test")
-            .setQuery(QueryBuilders.rangeQuery("field").gte("2010-01-03||+2d").lte("2010-01-04||+2d/d"))
-            .get();
-        assertNoFailures(searchResponse);
-        assertHitCount(searchResponse, 2L);
-
-        searchResponse = client().prepareSearch("test")
-            .setQuery(QueryBuilders.rangeQuery("field").gte("2010-01-05T02:00").lte("2010-01-06T02:00"))
-            .get();
-        assertNoFailures(searchResponse);
-        assertHitCount(searchResponse, 2L);
-
-        searchResponse = client().prepareSearch("test")
-            .setQuery(QueryBuilders.rangeQuery("field").gte("2010-01-05T02:00").lt("2010-01-06T02:00"))
-            .get();
-        assertNoFailures(searchResponse);
-        assertHitCount(searchResponse, 1L);
-
-        searchResponse = client().prepareSearch("test")
-            .setQuery(QueryBuilders.rangeQuery("field").gt("2010-01-05T02:00").lt("2010-01-06T02:00"))
-            .get();
-        assertNoFailures(searchResponse);
-        assertHitCount(searchResponse, 0L);
-
-        searchResponse = client().prepareSearch("test")
-            .setQuery(QueryBuilders.queryStringQuery("field:[2010-01-03||+2d TO 2010-01-04||+2d/d]"))
-            .get();
-        assertHitCount(searchResponse, 2L);
-    }
-
+    // TODO: Move this test to ParameterizedSimpleSearchIT after https://github.com/opensearch-project/OpenSearch/issues/8371
     public void testSimpleTerminateAfterCount() throws Exception {
```

Comment:
I realize the diff here is quite confusing but basically I parameterized the `SimpleSearchIT` class except for this test which we know is flakey from https://github.com/opensearch-project/OpenSearch/issues/9946

As a part of the follow-up for https://github.com/opensearch-project/OpenSearch/issues/8371 I will parameterize this test as well since the flakiness should be resolved with that.